applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -205,11 +203,22 @@ var Settingstitle = form.DummyValue.extend({
 	renderWidget: function() {
 		var body = E([
 			E('h3', 'Settings'),
-			E('body','First four options are mandatory. Also be sure to enable rrm reports, 80211kv, etc see: https://openwrt.org/docs/guide-user/network/wifi/usteer')
+			E('div','First four options are mandatory. Also be sure to enable rrm reports, 80211kv, etc see: https://openwrt.org/docs/guide-user/network/wifi/usteer'),
```

Comment:
Please revise: 
First four options are mandatory
can differ in meaning from
The first four options are mandatory

Have you tested this in-situ? I suspect that the 'link' there will not work. This might need revising.

Reply:
Text I can revise yes. That link is no link. Not sure if we wanted to have links there. Currently its just a pointer where to get info. I can make it either way.

Reply:
@Ramon00 I fixed the link in my PR https://github.com/Ramon00/luci/pull/2/commits/affb33d228adc6cc85409486d269c41611242ca0